cluster/vagrant/provision-minion.sh

```diff
@@ -72,16 +72,17 @@ EOF
 # See: https://github.com/mitchellh/vagrant/issues/2430
 hostnamectl set-hostname ${MINION_NAME}
 
-# Workaround to vagrant inability to guess interface naming sequence
-# Tell system to abandon the new naming scheme and use eth* instead
-rm -f /etc/sysconfig/network-scripts/ifcfg-enp0s3
```

Comment:
Or to ask another way, is this network script present when running on VMWare and required?  If so, do you know why?